### DISCUSSION TITLE
Cant install mariadb

### DISCUSSION THREAD

**Discussion by MUFASI** [ish-app/ish#1776] [Q&A]
Im trying to install 
![91330462-FA9F-46E2-8ED5-48E5EC7672CB](https://user-images.githubusercontent.com/103787016/163649583-529e3448-5db5-41a1-856e-91d175e6cc2a.png)
mariadb, but it isnt working.
I tried: “mariadbd-safe -u root &”
And it doesn’t work, the app just crashes. No error code, nothing. Just crash.
I managed to install mariadb and mariadb-client
As a matter effect i`m using a iPad 8th gen

**Comment by MUFASI** [comment]
> Im trying to install 
> 
> ![91330462-FA9F-46E2-8ED5-48E5EC7672CB](https://user-images.githubusercontent.com/103787016/163649583-529e3448-5db5-41a1-856e-91d175e6cc2a.png)
> 
> mariadb, but it isnt working.
> 
> I tried: “mariadbd-safe -u root &”
> 
> And it doesn’t work, the app just crashes. No error code, nothing. Just crash.
> 
> I managed to install mariadb and mariadb-client
> 
> As a matter effect i`m using a iPad 8th gen

It also crashes when i type 'mysqld' and 'mariadbd'

**Reply by saagarjha** [reply]
This might be #597?

**Reply by MUFASI** [reply]
> This might be #597?

So it will just keep crashing?
